### DISCUSSION TITLE
Option to use [_symbol_] as Kanban columns

### DISCUSSION THREAD

**Discussion by DillPickle80** [mgmeyers/obsidian-kanban#1056] [Ideas & Feature Requests]
I wondered if others would find it a good idea and if it would be hard to implement for your plugin to have an option to use the symbol between brackets of tasks (example: x in [x]) as Kanban columns. 

This way, rather then fetching the tasks from a dedicated Board file in which heading sections represents the status of my task, the plugin could fetch easily Tasks from files not formatted to have headings as columns; rather then moving my task to another heading section when my card is moved from left to right to finally end up in Done and then having to check the box on my own, moving my card from Not Started ([ ]) to In Progress([/]), to Done([x]) would directly change the in-bracket symbol from [ ] to [/] to [x], and in the end moving a Task to Done would no longer require an extra step of also checking the task.

**Comment by DillPickle80** [comment]
Considering making a fork of the plugin and doing this on my own, but ngl i'm a bit lost in the code and i'm sure a little hand from someone familiar with the repo would save a lot of time.

**Reply by vsme** [reply]
I did it. https://github.com/mgmeyers/obsidian-kanban/pull/1138
